apps/api/src/app/environments-v1/usecases/construct-framework-workflow/construct-framework-workflow.usecase.ts

```diff
@@ -199,7 +209,25 @@ export class ConstructFrameworkWorkflow {
 
     return foundWorkflow;
   }
+
+  private processSkipOption(controlValues: { [x: string]: unknown }, variables: FullPayloadForRender) {
+    const skipRules = controlValues.skip as RulesLogic<AdditionalOperation>;
+
+    // TODO: check if we need to add more validation here (is empty object, etc.)
+    if (!skipRules) {
```

Comment:
We can use lodash/isEmpty and cover all of the empty cases of an object ;)

```suggestion
    if (isEmpty(skipRules)) {
```